lib/di/home_binding.dart

```diff
@@ -51,6 +52,9 @@ class HomeBinding implements Bindings {
     final registratCubit = RegistrationCubit();
     Get.put(registratCubit, permanent: true);
 
+    final lenguageCubit = LanguageCubit();
```

Comment:
it quite unnecessary assignment? WDYT?

Reply:
@huynguyennovem This is needed for initialBinding

Reply:
@hoangdat Is it ok?